digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/middleware/TrafficSignManager.scala

```diff
@@ -0,0 +1,37 @@
+package fi.liikennevirasto.digiroad2.middleware
+
+
+import java.security.InvalidParameterException
+
+import fi.liikennevirasto.digiroad2.service.pointasset.{TrafficSignInfo, TrafficSignService}
+import fi.liikennevirasto.digiroad2.TrafficSignType
+import fi.liikennevirasto.digiroad2.service.linearasset.{ManoeuvreCreationException, ManoeuvreService, ProhibitionService}
+
+class TrafficSignManager(manoeuvreService: ManoeuvreService, prohibitionService: ProhibitionService) {
+
+  def trafficSignsCreateAssets(trafficSignInfo: TrafficSignInfo, newTransaction: Boolean = true ): Unit = {
```

Comment:
since the method is on TrafficSignManager, i don't think it's necessary to have trafficSigns included in the method name. could be renamed to just createAssets